test/test_jit.py

```diff
@@ -3223,6 +3223,19 @@ def foo(x):
         def tuple_call():
             return foo((1, 2))
 
+    def test_contiguous(self):
+        @torch.jit.script
+        def foo(x):
+            return 3 + x.contiguous()
+
+        x = torch.rand(1, dtype=torch.float, requires_grad=True)
```

Comment:
`x` is contiguous here, we want to test against both when `x` is /isn't contiguous I think?